packages/editor/src/browser/editor-navigation-contribution.ts

```diff
@@ -88,6 +96,14 @@ export class EditorNavigationContribution implements Disposable, FrontendApplica
         this.toDispose.dispose();
     }
 
+    /**
+     * Toggle the display of minimap in the editor.
+     */
+    protected async doToggleMinimap(): Promise<void> {
+        const value: boolean | undefined = this.preferenceService.get('editor.minimap.enabled');
+        this.preferenceService.set('editor.minimap.enabled', !value);
```

Comment:
It might be better to be explicit and state the scope: `PreferenceScope.User`.

Reply:
Thanks! Just fixed.